resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/Config.scala

```diff
@@ -24,6 +24,14 @@ import org.apache.spark.internal.config.ConfigBuilder
 
 private[spark] object Config extends Logging {
 
+    val DECOM_SCRIPT =
+    ConfigBuilder("spark.kubernetes.decom.script")
```

Comment:
Maybe `decom` -> `decommission`? Looks like other configuration names don't use such abbreviation .

Reply:
+1 for @HyukjinKwon 's suggestion. Ya, right. We use full name there, `spark.storage.decommission.enabled`.

Reply:
Sounds good :)